src/components/TheConsumptionParameters.vue

```diff
@@ -68,7 +70,15 @@ function setSavings(savingsPercent: number) {
             :min="1"
             :step="1000"
             :validator="validatePositive"
-            :on-input="emit('consumptionChanged', model.consumption)"
+            :on-input="
+              debounce(
+                () => {
+                  trackInput('consumption') &&
+                  emit('consumptionChanged', model.consumption)
+                },
+                2000
+              )
+            "
```

Comment:
I understand, that the`debounce()` function is a HOF, thus returning a function, so that might work at least.

However, according to https://www.naiveui.com/en-US/os-theme/components/input-number there is no such property `on-input`.

Maybe `on-update:value` would work?

Reply:
Ok, this has been som trial-and-lots-of-errors.
tl;dr: I always wanted to use `on-update:value` but this lead to the input not reacting on interactions with the +/-
[`on-input`](https://www.naiveui.com/en-US/os-theme/components/input#Input-Props) was triggered however.

Bottom line: using `ref` and reactive state passed as props requires props to be watched explicitly (see [this comment](https://github.com/mrsimpson/energy/pull/89/commits/da3fc51397144f3805959e32e0c7ec7765e88034#diff-a104f881632b86b395547ee5d44e6e4fb8dba269f55e8eae4c11360c6caf1399R25-R27))
At least now, it looks fine to me